docs/user/guides/subprojects.rst

```diff
@@ -0,0 +1,57 @@
+How to manage subprojects
+=========================
+
+This guide shows you how to manage subprojects,
+which is a way to host multiple projects under a "main project".
+
+.. seealso::
+
+   :doc:`/subprojects`
+      Read more about what the subproject feature can do and how it works.
+
+Adding a subproject
+-------------------
+
+In the admin dashboard for your project, select :guilabel:`Subprojects` from the left menu.
+From this page you can add a subproject by choosing a project from the :guilabel:`Subproject` dropdown
+and typing an alias in the :guilabel:`Alias` field.
+
+Immediately after adding the subproject, it will be hosted at the URL displayed in the updated list of subprojects.
+
+.. image:: /img/screenshot_subprojects_list.png
```

Comment:
Re: @humitos comment about HowTo's, screenshots also don't match across .com & .org currently. It will be better once we ship the new templates.

Reply:
Ah yeah, maybe with tabbed panels, we can find a clever way to fix this in the future.

It's okay to stick with .org screenshots for now, right? Pretty sure that's the general pattern?

Reply:
Yea, it's fine.